TipCatDotNet.Api/Services/Payments/TransactionService.cs

```diff
@@ -44,7 +45,37 @@ public async Task<Result> Add(PaymentIntent paymentIntent, string? message, Canc
         _context.Transactions.Add(newTransaction);
         await _context.SaveChangesAsync(cancellationToken);
 
+        var (_, isFailure, error) = await SetPaymentTime(memberId);
+        if (isFailure)
+            return Result.Failure(error);
+
         return Result.Success();
+
+
+        async Task<Result> SetPaymentTime(int memberId)
+        {
+            var stripeAccount = await _context.StripeAccounts
+                .SingleOrDefaultAsync(sa => sa.MemberId == memberId, cancellationToken);
+
+            if (stripeAccount == null)
```

Comment:
Not a bug, but could you write `is null` instead of `== null`? For better readability